modules/setting-up-cpu-manager.adoc

```diff
@@ -9,21 +9,19 @@
 
 .Procedure
 
-. Optional: Label a node:
+. Label a node by running the following command:
 +
 [source,terminal]
 ----
 # oc label node perf-node.example.com cpumanager=true
 ----
 
-. Edit the `MachineConfigPool` of the nodes where CPU Manager should be enabled. In this example, all workers have CPU Manager enabled:
```

Comment:
The right hand side implies all, the left implies just a few are assigned the CPUManager configurations. I think the left while worded strangely, is accurate to what you can do.

Reply:
Paul thanks for reviewing it. 
We are trying to remove the optional keyword in this step because it is a must when we execute the example verification in later steps. Therefore one of the node must be labeled here and CPU manager will be enabled on all worker nodes.
Could you please explain if i misunderstood you?